sdk/resources/mgmt/src/main/java/com/azure/management/RestClientBuilder.java

```diff
@@ -50,7 +51,7 @@
      */
     public RestClientBuilder() {
         retryPolicy = new RetryPolicy();
-        httpLogOptions = new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS);
+        httpLogOptions = new HttpLogOptions().setLogLevel(HttpLogDetailLevel.NONE);
```

Comment:
I've changed the default level to NONE.

1. it is consistent with track1
2. I think NONE should be the default.

Reply:
interesting that track1 default NONE..

Reply:
Always a potential security risk if print out too many things. It is less risky for track2 log, since by default only a few header/query is white-listed.